src/vm/debugdebugger.cpp

```diff
@@ -796,52 +801,52 @@ FCIMPL4(void, DebugStackTrace::GetStackFramesInternal,
                     OBJECTREF obj = (OBJECTREF) StringObject::NewString(wszFileName);
                     pStackFrameHelper->rgFilename->SetAt(iNumValidFrames, obj);
                 }
-            }
 #endif // FEATURE_ISYM_READER
 
-            // If the above isym reader code did NOT set the source info either because it is ifdef'ed out (on xplat)
-            // or because the pdb is the new portable format on Windows then set the information needed to called the
-            // portable pdb reader in the StackTraceHelper. The source/line info isn't valid on ENC'ed modules.
-            if (!fFileInfoSet && !fIsEnc)
-            {
-                // Save MethodToken for the function
-                I4 *pMethodToken = (I4 *)((I4ARRAYREF)pStackFrameHelper->rgiMethodToken)->GetDirectPointerToNonObjectElements();
-                pMethodToken[iNumValidFrames] = pMethod->GetMemberDef();
+                // If the above isym reader code did NOT set the source info either because it is ifdef'ed out (on xplat)
+                // or because the pdb is the new portable format on Windows then set the information needed to called the
```

Comment:
Nit `set the information needed to called` -> `set the information needed to call`